src/pages/journal/components/test-outcome/test-outcome.ts

```diff
@@ -47,22 +49,18 @@ export class TestOutcomeComponent {
     return this.canSubmitTest;
   }
 
-  startTest() {
-    if (startsWith(this.slotId.toString(), end2endPracticeSlotId)) {
-      this.store$.dispatch(new StartE2EPracticeTest(this.slotId.toString()));
-    } else {
-      this.store$.dispatch(new StartTest(this.slotId));
-    }
-    this.navController.push(COMMUNICATION_PAGE);
-  }
-
   writeUpTest() {
-    this.store$.dispatch(new ActivateTest(this.slotId));
+    this.store$.dispatch(new ActivateTest(this.slotDetail.slotId));
     this.navController.push(OFFICE_PAGE);
   }
 
   resumeTest() {
-    this.store$.dispatch(new ActivateTest(this.slotId));
+    if (this.showRekeyModal()) {
```

Comment:
the name of this method is a bit too similar to `displayRekeyModal`. They're a bit confusing when the appear together like this if statement. Can we rename so they're distinct?